spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/util/FormatUtil.java

```diff
@@ -1,13 +1,18 @@
 package capital.scalable.restdocs.util;
 
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
 import static org.springframework.util.StringUtils.arrayToDelimitedString;
 
-public class ObjectUtil {
-    private ObjectUtil() {
+public class FormatUtil {
+    private FormatUtil() {
         // util
     }
 
     public static String arrayToString(Object[] o) {
         return "[" + arrayToDelimitedString(o, ", ") + "]";
     }
+
+    public static String addDot(String text) {
+        return isNotBlank(text) && !text.endsWith(".") ? text + "." : text;
```

Comment:
Are we already trimming the text before this? If not, it would be good to do it here.

Reply:
ClassJavadoc always trimms or returns empty string.